tests/testthat/test_agg.R

```diff
@@ -0,0 +1,346 @@
+library(data.table)
+library(testthat)
+
+# Aggregate categorical variable with varying interval id cols ------------
+
+# Inputs:
```

Comment:
I like this, especially for more complicated testing and/or scripts with more tests. Did you add this to a style guide somewhere?

Reply:
Good idea, added a section here about "Documenting tests" https://github.com/ihmeuw-demographics/packageTemplate/wiki/Creating-tests